js/libs/form.js

```diff
@@ -436,16 +436,19 @@ Fliplet.Widget.instance('form-builder', function(data) {
 
         if (data.autobindProfileEditing) {
           return Fliplet.Session.get().then(function (session) {
+            var isEditMode = false;
```

Comment:
how can line 88 access this variable?

please define the `var` declaration at the top of the file and only assign its value when necessary (e.g. on this line, instead of declaring the var)